tests/tests.rs

```diff
@@ -44,6 +45,36 @@ fn run_mode<F: Fn(&mut compiletest_rs::Config)>(
     compiletest_rs::run_tests(&config);
 }
 
+/// Builds LLVM bitcode files from LLVM IR files located in a specified directory.
+fn build_bitcode<P>(dir: P)
+where
+    P: AsRef<Path>,
+{
+    for entry in fs::read_dir(dir.as_ref()).expect("failed to read the directory") {
+        let entry = entry.expect("failed to read the file");
+        let path = entry.path();
+
+        if path.is_file() && path.extension().unwrap_or_default() == "ll" {
+            let bc_dst = path.with_extension("bc");
```

Comment:
this builds in the source tree, while it should probably build in the target
directory? (where does compiletest put artifacts?)

Reply:
Compile test puts artifacts in /tmp.

I'm not sure whether I can pass environment variables to `compile-flags`. If I can, I could try to use `OUT_DIR` or some other variable coming from cargo to put the artifacts in target/. Otherwise, I think putting them in /tmp would be the easiest.